interfaces/contracts/IAssetMatcher.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import "@rarible/lib-asset/contracts/LibAsset.sol";
+
+interface IAssetMatcher {
```

Comment:
@SKonstantinS this code duplicates `exchange-v2/contracts/IAssetMatcher`. Let's better keep the newly added one, and remove the old one?

Reply:
@serejke agree with you, early I made a task to delete IAssetMatcher from exchange-v2/contracts/IAssetMatcher: https://rarible.atlassian.net/browse/RPN-957